AutocompleteClient/FW/Config/AutocompleteConfig.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Constants.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+public struct AutocompleteConfig {
+
+    public let autocompleteKey: String
+    public var resultCount: AutocompleteResultCount?
+    public var clientID: String?
+    
+    public init(autocompleteKey: String, resultCount: AutocompleteResultCount? = nil, clientID: String? = nil){
+        self.autocompleteKey = autocompleteKey
+        self.resultCount = resultCount
+        self.clientID = clientID
```

Comment:
I think there's a misunderstanding of what the Client ID is... The Client ID passed to the server in the i parameter should be a randomly generated GUID that is not configurable by the customer and is the same for all requests from a given app.